spec/jobs/workspace_details_job_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe WorkspaceDetailsJob, type: :job do
+  let!(:user) { create(:user) }
+  let!(:workspace) { create(:workspace, user: user) }
+  let!(:file) { WickedPdf.new.pdf_from_string('<h1>Hello There!</h1>') }
+
+   it "get a details" do
+    ActiveJob::Base.queue_adapter = :test
+    expect { WorkspaceDetailsJob.perform_later(workspace, user) }.to have_enqueued_job
+  end
+
+  it "run mailer" do
+    ActiveJob::Base.queue_adapter = :test
+    expect {WorkspacesMailer.workspace_details(workspace, user, file).deliver_later}.to have_enqueued_job.on_queue('mailers')
```

Comment:
do similar test as above
BUT
chech that mailer method was executed (you did similar before)